client/src/selectPinboard.tsx

```diff
@@ -459,16 +481,21 @@ export const SelectPinboard = ({
           </div>
         </div>
         {loading && searchText && <Text>Searching...</Text>}
+        {data && searchText && (
+          <Text>
+            {unopenedPinboards.length > 0
+              ? "Shift + click opens in Composer"
```

Comment:
Would it be worth including this note under the "MY TEAMS' PINBOARDS" header as well as the "SEARCH" header? As the functionality seems to apply to both sections

Reply:
astute observation. I did consider that but its only directly useful for the search results, and the fact the same behaviour applies to the 'my pinboards' and 'my teams pinboards' is because they share a component, and I'm not sure it warrants passing in as a prop whether this new shift behaviour should apply

Reply:
Yup, sounds reasonable!